core/controller/src/main/scala/org/apache/openwhisk/core/controller/Activations.scala

```diff
@@ -72,6 +74,8 @@ object WhiskActivationsApi {
 /** A trait implementing the activations API. */
 trait WhiskActivationsApi extends Directives with AuthenticatedRouteProvider with AuthorizedRouteProvider with ReadOps {
 
+  protected val disableStoreResultConfig: Boolean = loadConfig[Boolean](ConfigKeys.disableStoreResult).getOrElse(false)
```

Comment:
I think most other configs use the convention that a) defaults are in the application.conf and b) no defaults are in the code. Also I think `loadConfigOrThrow()` is preferred.